OpenTabletDriver.Desktop/Settings.cs

```diff
@@ -143,12 +143,19 @@ void propertyWatch(object _, PropertyChangedEventArgs p)
 
         public void Serialize(FileInfo file)
         {
-            if (file.Exists)
-                file.Delete();
+            try
+            {
+                if (file.Exists)
+                    file.Delete();
 
-            using (var sw = file.CreateText())
-            using (var jw = new JsonTextWriter(sw))
-                serializer.Serialize(jw, this);
+                using (var sw = file.CreateText())
+                using (var jw = new JsonTextWriter(sw))
+                    serializer.Serialize(jw, this);
+            }
+            catch (System.UnauthorizedAccessException)
```

Comment:
Same situation here, add a using statement.
```suggestion
            catch (UnauthorizedAccessException)
```